go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	github.com/sirupsen/logrus v1.4.1
 	github.com/spf13/cobra v0.0.0-20160830174925-9c28e4bbd74e
 	github.com/spf13/pflag v1.0.5
-	github.com/stellar/go v0.0.0-20230623175925-531797f01226
+	github.com/stellar/go v0.0.0-20230613070852-e20e8bc1619c
```

Comment:
@sreuland this is an older branch.. I think that for the time being we should leave it with tip of tree of soroban-xdr-next-next on the go monorepo.

Reply:
oh, great catch, my mistake, I updated that to soroban-xdr-next by mistake, I updated this ref to use soroban-xdr-next-next to your point and that resolved the other xdr compile errors also.